src/openrct2/world/TileElement.cpp

```diff
@@ -170,3 +171,12 @@ EntranceElement* TileElementBase::AsEntrance()
 {
     return as<EntranceElement>();
 }
+
+const WallElement* TileElementBase::AsWall() const
+{
+    return as<WallElement>();
+}
+WallElement* TileElementBase::AsWall()
+{
+    return as<WallElement>();
+}
```

Comment:
This is moved to `TileElementBase.cpp` in my PR (along with all the other ‘As’ functions).